### PR TITLE
PD-1196: Adds Get Help dropdown to MongoNav

### PR DESCRIPTION
**Pull request by brookescarlett**
🎟 _Jira ticket:_ [PD-1196](https://jira.mongodb.org/browse/PD-1196)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: packages/mongo-nav/src/org-nav/OrgNav.tsx [hunk 1]
```diff
@@ -46,17 +47,24 @@ const navContainer = css`
...
-    marginLeft: ['16px', '16px', '24px'],
   })}
 `;
 
-const rightLinkMargin = css`
-  ${mq({
-    marginRight: ['16px', '16px', '24px'],
-  })}
+const externalIconStyles = css`
+  &:hover {
```
Comment (DesignerDave) [inline packages/mongo-nav/src/org-nav/OrgNav.tsx:57]: This selector can be simplified:
```
&:hover svg {
  color: ${uiColors.blue.dark2};
}
```

File: packages/mongo-nav/src/org-nav/OrgNav.tsx [hunk 2]
```diff
@@ -115,46 +123,90 @@ function DropdownMenuIcon({ open }: { open: boolean }) {
...
+          onClick={onElementClick(NavElement.OrgNavDocs)}
+        >
+          <div
+            className={css`
+              display: flex;
+              align-items: center;
+              justify-content: space-between;
+            `}
+          >
+            Docs{' '}
```
Comment (DesignerDave) [inline packages/mongo-nav/src/org-nav/OrgNav.tsx:186]: Is that empty string for spacing? We shouldn't do that if so.

File: packages/mongo-nav/src/org-nav/OrgNav.tsx [hunk 3]
```diff
@@ -115,46 +123,90 @@ function DropdownMenuIcon({ open }: { open: boolean }) {
...
+        `}
+      >
+        <MenuItem
+          href={urls.support}
+          data-testid="org-nav-support-link"
+          active={activeNav === NavElement.OrgNavSupport}
+          size={isTablet ? 'large' : 'default'}
+          onClick={onElementClick(NavElement.OrgNavSupport)}
+        >
+          <div>Support</div>
```
Comment (DesignerDave) [inline packages/mongo-nav/src/org-nav/OrgNav.tsx:170]: Why is this div here?

File: packages/mongo-nav/src/org-nav/OrgNav.tsx [hunk 4]
```diff
@@ -115,46 +123,90 @@ function DropdownMenuIcon({ open }: { open: boolean }) {
...
+        usePortal={false}
+        className={css`
+          width: 128px;
+        `}
+      >
+        <MenuItem
+          href={urls.support}
+          data-testid="org-nav-support-link"
+          active={activeNav === NavElement.OrgNavSupport}
+          size={isTablet ? 'large' : 'default'}
```
Comment (DesignerDave) [inline packages/mongo-nav/src/org-nav/OrgNav.tsx:167]: What about mobile?

File: packages/mongo-nav/src/org-nav/OrgNav.tsx (hunk 2 above)
Comment (DesignerDave) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx:186]: Also the new alignment probably makes this unnecessary anyways.

File: packages/mongo-nav/src/org-nav/OrgNav.tsx [hunk 5]
```diff
@@ -115,46 +123,90 @@ function DropdownMenuIcon({ open }: { open: boolean }) {
...
+          <div>Support</div>
+        </MenuItem>
+        <MenuItem
+          href={urls.docs}
+          data-testid="org-nav-docs-link"
+          size={isTablet ? 'large' : 'default'}
+          className={externalIconStyles}
+          onClick={onElementClick(NavElement.OrgNavDocs)}
+        >
+          <div
```
Comment (DesignerDave) [inline packages/mongo-nav/src/org-nav/OrgNav.tsx:179]: What's this div for? Can the styles on this div live on the menu item?

File: packages/mongo-nav/src/org-nav/OrgNav.tsx [hunk 6]
```diff
@@ -185,44 +237,26 @@ function NavLinks({
...
-        </VersionNumber>,
-      );
-    } else {
-      items.unshift(
-        <VersionNumber style={VersionNumberStyle.Nav}>
-          {onPremVersion}
-        </VersionNumber>,
-      );
-    }
+    items.unshift(
```
Comment (DesignerDave) [inline packages/mongo-nav/src/org-nav/OrgNav.tsx:255]: This seems more clever than it needs to be. Can we just write this conditional and element within the JSX instead to ensure the correct ordering?

**Review by DesignerDave — COMMENTED** [review]
A few notes on this, but it's looking a lot better than the last iteration.

As an aside, I'm not seeing cursor: pointer being applied to the menu items. This bug seems to exist on main as well, but we should probably address it regardless. Should be a one-liner.

File: packages/mongo-nav/src/org-nav/OrgNav.tsx (hunk 1 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx:57]: 👍

File: packages/mongo-nav/src/org-nav/OrgNav.tsx (hunk 2 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx:186]: 👍

File: packages/mongo-nav/src/org-nav/OrgNav.tsx (hunk 3 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx:170]: Good question, it's gone now.

File: packages/mongo-nav/src/org-nav/OrgNav.tsx (hunk 4 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx:167]: isTablet just means is at least tablet, so mobile is included here

File: packages/mongo-nav/src/org-nav/OrgNav.tsx (hunk 5 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx:179]: the flex styles were getting overwritten when this is moved up

File: packages/mongo-nav/src/org-nav/OrgNav.tsx (hunk 6 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/org-nav/OrgNav.tsx:255]: 👍

**Review by DesignerDave — APPROVED** [review]

**Pull request merged**